pom.xml

```diff
@@ -82,15 +82,21 @@
 		</dependency>
 		
 		<dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-websocket</artifactId>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-websocket</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.eclipse.paho</groupId>
+			<artifactId>org.eclipse.paho.client.mqttv3</artifactId>
```

Comment:
Do you plan to implement some methods into ArrowheadService.calss for mqtt?

Reply:
That's the plan. I will implement broker management and methods for interacting with the core systems.